inference-engine/thirdparty/clDNN/src/graph_optimizer/reorder_inputs.cpp

```diff
@@ -575,14 +576,32 @@ void reorder_inputs::run(program& p, layout_optimizer& lo, reorder_factory& rf)
         }
     };
 
+    const auto reorder_input_fully_connected = [&p, &lo, &rf](typed_program_node<fully_connected>& fc_node) {
+        auto& weights = fc_node.weights();
+        auto& input = fc_node.input();
+        auto weights_layout = weights.get_output_layout();
+        auto input_layout = input.get_output_layout();
+        // Change input data of fully-connected node from bx to bf
+        if (format::is_simple_data_format(weights_layout.format) && format::is_simple_data_format(input_layout.format) && weights.is_constant() &&
```

Comment:
Is it necessary to check `weights_layout`?

Reply:
Removed